etc/ci_scripts/dump_artifacts/analysis/pipeline_message_flow.py

```diff
@@ -0,0 +1,103 @@
+import json
+import re
+from pathlib import Path
+from typing import List, Optional
+
+# Declares the order in which to print diagnoses out.
+# Models the order of messages flowing through the pipeline.
+PRIORITIZATION = [
+    "sysmon-generator.stdout",
+    "osquery-generator.stdout",
+    "node-identifier.stdout",
+    "node-identifier-retry.stdout",
+    "graph-merger.stdout",
+    "analyzer-dispatcher.stdout",
+    "analyzer-executor.stdout",
+    "engagement-creator.stdout",
+]
+
+
+def analyze_grapl_core(artifacts_dir: Path, analysis_dir: Path) -> None:
+    logs_in_grapl_core = list((artifacts_dir / "grapl-core").iterdir())
```

Comment:
wtf python overrides / for paths I take it? lol

Reply:
Yeah. C++ too actually